pkg/conduit/pipelines.yml

```diff
@@ -0,0 +1,71 @@
+---
```

Comment:
Another thing we might want to add is a field like `version` which would right now just be `version: '1'` to already make it possible to evolve this schema in the future. Essentially the same as [docker-compose](https://docs.docker.com/compose/compose-file/compose-versioning/) does it. WDYT?

Reply:
@lovromazgon does it have to be the same version for all the yaml docs? cuz if not, then we can't merge all the docs in one map, since they could have different versions. WDYT?

Reply:
we also should decide on what the default value for the version should be, when it's not provided

Reply:
Right now we will only have a single parser, which only knows how to parse the config file of version 1. If the version string contains something else than `1` or `1.0` (I think it would make sense to have minor and major versions) the parser should probably fail because it can't be sure it can parse that document.

In the future when we create a new schema version we would change the parser and allow additional values in the version field (e.g. `1.1`) or create a new parser alongside the existing one (e.g. if we bump the major version to `2`), and some logic around that. But let's not worry about that, we can do all of this when/if that ever happens. For now we can assume we have this one parser and we can merge all parsed documents.

> what the default value for the version should be

Why not make the field mandatory? I mean the parser shouldn't care, but in the second step we'll need to validate whatever the parser read, and at that point we would raise an error if the `version` field is not specified.